src/api/spec/factories/bs_requests.rb

```diff
@@ -196,8 +196,46 @@
       end
     end
 
-    factory :bs_request_with_maintenance_incident_action do
+    factory :bs_request_with_maintenance_incident_actions do
       type { :maintenance_incident }
+
+      transient do
+        source_project_name { '' }
+        source_package_names { [] }
+        target_project_name { '' }
+        target_releaseproject_names { [] }
+      end
+
+      callback(:before_create) do |instance, evaluator|
+        buffer = []
+        evaluator.creator.run_as do
+          evaluator.source_package_names.each do |source_package_name|
+            evaluator.target_releaseproject_names.each do |target_releaseproject_name|
+              # TODO: find a better way to find out if the request comes from a branched project or and official project
+              # i.e. 'cacti.openSUSE_Leap_15.4_Update'
+              package_name = source_package_name
+              package_name += ".#{target_releaseproject_name.tr(':', '_')}" if source_project_name.starts_with?('home:')
```

Comment:
```suggestion
              package_name += ".#{target_releaseproject_name.tr(':', '_')}" if evaluator.source_project_name.starts_with?('home:')
```

Otherwise I see:
```
rake aborted!                               
NameError: undefined local variable or method `source_project_name' for #<FactoryBot::SyntaxRunner:0x00007f155cd9a058>
                                               
              package_name += ".#{target_releaseproject_name.tr(':', '_')}" if source_project_name.starts_with?('home:')
                                                                               ^^^^^^^^^^^^^^^^^^^
```

Reply:
Fixed